citi-onboarding/Aqua-Jr
# Hotfix/nav-links

## What I did
- Made the buttons in the nav change style depending on the section
- Animated the navbar so it changes style when scrolling
- Fixed the bug that the buttons in the nav wouldn't take the user to the right place

## How to test
1 - open the website
2 - Scroll down the page and check (in both versions):
  2.1 - A shadow is added to the nav when you scrow down
  2.2 - The buttons change color correctly depending on the current section
  2.3 - The buttons in the nav take you to the right place
